config/default/translations/messages-en.properties

```diff
@@ -0,0 +1,1648 @@
+setup.contact.title = Identify care coordinators
+setup.contact.description = Select who will be registering and tracking pregnancies in your community. They will need a mobile phone with a texting plan.
+setup.contact.help = If you don't know what to choose, refer to the ANC user guide.
+setup.language.title = Language preference
+setup.language.subtitle = Select the default language that website users of Medic Mobile will be using. Users can set individual preferences after logging in.
+setup.language.outgoing.subtitle = Select the language in which your care coordinators would like to receive automated messages from Medic Mobile.
+setup.language.description = If you don't see your language, you can always add a custom language from Administrative Settings.
+setup.registration.title = Registration form
+setup.registration.description = Will your care coordinators be reporting on Last Menstrual Period (LMP)? Women in some cultures may not want to share this information.
+setup.registration.help = If you don't know what to choose, refer to the ANC user guide.
+setup.skip = Skip the setup wizard
+setup.start = Finish
+tour.select = Select tour
+tour.select.description = You can start a tour at any time by clicking the menu icon in the top right corner of the page. Which tour would you like to run?
+Gateway\ number = Gateway Phone Number
+Gateway\ number\ help = This is number where mobile reporters should send their reports. It is also the number they will receive messages from.
+Default\ country\ code = Default country code
+Default\ country\ code\ help = This code will be attributed to all phone numbers entered into the system without a country code.
+Pregnant\ patient = The pregnant patient
+Facility\ nurse = Facility nurse
+with.lmp = With Last Menstrual Period (LMP)
+without.lmp = Without Last Menstrual Period (LMP)
+Error\ saving\ settings = There was an error saving your settings, please try again.
+Generated\ report\ field = Generated report field
+birth_date = Birth date
+mother_outcome = Mother outcome
+child_birth_outcome = Child birth outcome
+child_birth_weight = Child birth weight
+child_birth_date = Child birth date
+expected_date = Expected date
+Any\ status = Any status
+Patient\ Name = Patient name
+Weeks\ Pregnant = Weeks pregnant
+Visits = Visits
+contact.short = CHW
+EDD = EDD
+Appointment\ Date = Appointment date
+Patient\ History = Patient history
+Last\ Appointment = Last appointment
+analytics.unconfigured = No analytics modules are configured.
+analytics.connection.error = Could not retrieve data at this time.
+analytics.anc.active-pregnancies = Active Pregnancies
+analytics.anc.delivery-locations = Reported Delivery Locations
+analytics.anc.high-risk = High Risk Pregnancies
+analytics.anc.missed-appointments = Recent Missed Appointments
```

Comment:
Are all these basic translation keys required? These don't seem related to this config - aren't they already included in the app?

Reply:
medic-conf merges any translations you have here with the existing translations in the webapp. You shouldn't have to redefine any, especially since this is going to be the default configuration for webapp. It would be confusing if there were two default translations for everything.

Reply:
The labels included with the app automatically are in [ddocs/medic/_attachments/translations/messages-en.properties](https://github.com/medic/medic/blob/master/ddocs/medic/_attachments/translations/messages-en.properties), so you'll want to include in this config only the ones that are additional or different.

Reply:
Done.